### PR TITLE
Added .NET 5 App to integration tests

### PR DESCRIPTION
**Pull request by cijothomas**
Adding .NET 5 to the integration tests. Another step in direction of support .NET 5.

This just lays the ground work for testing. As a next step, we can probably remove all the three separate test projects and consolidate them into a single one with multitarget. Or atleast move controller classes as shared files.

### Checklist
- [ ] I ran Unit Tests locally.
- [ ] CHANGELOG.md updated with one line description of the fix, and a link to the original issue if available.

For significant contributions please make sure you have completed the following items:

- [ ] Design discussion issue #
- [ ] Changes in public surface reviewed

The PR will trigger build, unit tests, and functional tests automatically. Please follow [these](https://github.com/Microsoft/ApplicationInsights-dotnet/blob/develop/.github/CONTRIBUTING.md) instructions to build and test locally.

### Notes for authors:
- FxCop and other analyzers will fail the build. To see these errors yourself, compile localy using the Release configuration.

### Notes for reviewers:

- We support [comment build triggers](https://docs.microsoft.com/azure/devops/pipelines/repos/github?view=azure-devops&tabs=yaml#comment-triggers)
  - `/AzurePipelines run` will queue all builds
  - `/AzurePipelines run <pipeline-name>` will queue a specific build

### CODE REVIEW

File: NETCORE/.vsts/linux-build.yml [hunk 1]
```diff
@@ -21,6 +23,12 @@ steps:
   inputs:
     version: 3.1.301
     
+- task: UseDotNet@2
+  displayName: install dotnet core 5
```
Comment (cijothomas) [inline NETCORE/.vsts/linux-build.yml:27]: @eddynaka Its donet 5, not dotnet core 5 :D

Comment (eddynaka) [reply NETCORE/.vsts/linux-build.yml:27]: updated!

**Comment by TimothyMothra** [comment]
This PR is starting to get very large.
Could you submit a separate PR for adding the "IntegrationTests.WebApp.5.0" project?
I think that would shrink this PR by almost half.

**Comment by eddynaka** [comment]
> This PR is starting to get very large.
> Could you submit a separate PR for adding the "IntegrationTests.WebApp.5.0" project?
> I think that would shrink this PR by almost half.

Hi @TimothyMothra , I think the best way to go is:
1. create one separate PR just updating the pipeline and the nugets, so everything needs to be fine.
2. merge the (1) into this, then we will have only net5.

Let me do that later today and i will mark both (you and cijo)

**Comment by eddynaka** [comment]
> > This PR is starting to get very large.
> > Could you submit a separate PR for adding the "IntegrationTests.WebApp.5.0" project?
> > I think that would shrink this PR by almost half.
> 
> Hi @TimothyMothra , I think the best way to go is:
> 
> 1. create one separate PR just updating the pipeline and the nugets, so everything needs to be fine.
> 2. merge the (1) into this, then we will have only net5.
> 
> Let me do that later today and i will mark both (you and cijo)

(1) is done
(2) just merged, so we just have files that are used to net5.

Not sure if we want to add windows in the linux pipeline. Let me know, i can remove those.

**Review by TimothyMothra — APPROVED** [review]

**Pull request merged**